spock-specs/src/test/groovy/org/spockframework/smoke/extension/TimeoutExtension.groovy

```diff
@@ -29,13 +29,12 @@ import static java.util.concurrent.TimeUnit.*
  * @author Peter Niederwieser
  */
 class TimeoutExtension extends EmbeddedSpecification {
-  @Shared Thread testFrameworkThread = Thread.currentThread()
 
   def setup() {
     runner.addClassMemberImport TimeUnit
   }
 
-  @Timeout(1)
+  @Timeout(10)
```

Comment:
As the tests are right now, I can only see that the `@Timeout` extension does not really work with the new changes if we have to increase the timeouts by a factor of 10.